Private/Find-ESC8.ps1

```diff
@@ -8,26 +8,20 @@
         $ADCSObjects | Where-Object {
             $_.CAEnrollmentEndpoint
         } | ForEach-Object {
-            $Issue = [ordered] @{
-                Forest            = $_.CanonicalName.split('/')[0]
-                Name              = $_.Name
-                DistinguishedName = $_.DistinguishedName
+            $Issue = [pscustomobject]@{
+                Forest               = $_.CanonicalName.split('/')[0]
+                Name                 = $_.Name
+                DistinguishedName    = $_.DistinguishedName
+                CAEnrollmentEndpoint = $_.CAEnrollmentEndpoint
+                Issue                = 'HTTP enrollment is enabled.'
+                Fix                  = '[TODO]'
+                Revert               = '[TODO]'
+                Technique            = 'ESC8'
             }
-            if ($_.CAEnrollmentEndpoint -like '^http*') {
-                $Issue['Issue'] = 'HTTP enrollment is enabled.'
-                $Issue['CAEnrollmentEndpoint'] = $_.CAEnrollmentEndpoint
-                $Issue['Fix'] = 'TBD - Remediate by doing 1, 2, and 3'
-                $Issue['Revert'] = 'TBD'
-            } else {
+            if ($_.CAEnrollmentEndpoint -like '^https*') {
                 $Issue['Issue'] = 'HTTPS enrollment is enabled.'
-                $Issue['CAEnrollmentEndpoint'] = $_.CAEnrollmentEndpoint
-                $Issue['Fix'] = 'TBD - Remediate by doing 1, 2, and 3'
-                $Issue['Revert'] = 'TBD'
             }
-            $Issue['Technique'] = 'ESC8'
-            $Severity = Set-Severity -Issue $Issue
-            $Issue['Severity'] = $Severity
-            [PSCustomObject] $Issue
+            [PSCustomObject]$Issue
```

Comment:
Curious why you added the explicit type for this function, but not the others? (Other than the fact that this existed in the previous version of this function as well.) :)